design.md

```diff
@@ -2,12 +2,28 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Our goal is to make the syntax less intimidating for computer novices. We want to turn picobot from a coding problem into a logic puzzle, where instead of writing code that controls the robot, it feels more like users are writing instructions. We're trying to assume as little knowledge as possible on the part of the user.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+By removing the code flavor, we expand the domain of maze-searching to something that those without a coding background can do. The hope is that writing a program in this language doesn't feel like "coding" in a traditional sense. Maze navigation has problems in it that aren't necessarily best solved by the kind of mindset that you'd have if you feel you're writing code or a program -- e.g., repeating a pattern over and over (but not necessarily thinking of loops or states). In particular, we want to get away from the idea of states which is so deeply a part of picobot. We've essentially replaced states with goto statements.
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+In theory, the behaviours that are needed to solve these puzzle should be inherently easier to express. Firstly, the code is read sequentially, which we believe is easier to think about than states and rules. Additionally, the user is only expected to provide the necessary information at each line, i.e. only the state conditions that are blocked or open (not "anything"). We also hope that the use of natural language is appropriate for different audiences than the original picobot design.
+
```

Comment:
I agree with Milo as sacrificing the states might make it more difficult to understand / read the code.